example/presto_cluster/main.tf

```diff
@@ -132,7 +132,7 @@ module "postgres" {
 }
 
 module "hive" {
-  source = "github.com/fredrikhgrelland/terraform-nomad-hive.git?ref=0.3.0"
+  source = "github.com/fredrikhgrelland/terraform-nomad-hive.git?ref=0.3.1"
```

Comment:
Do you plan to wait for hive release 0.3.1? I would recommend do not bump versions in release related PR (this PR)

Reply:
I was planning to wait for hive-release, yes. I agree it shouldn't be done in the same PR, but thought it might be ok in this case because this is basically a hotfix to fix the docker_pull_timeout problem